### PR TITLE
do not run ceph-rest-api on nautilus and above

### PR DESCRIPTION
**Pull request by leseb**
The package has been removed.

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

File: src/daemon/demo.sh [hunk 1]
```diff
@@ -381,7 +381,12 @@ function build_bootstrap {
         bootstrap_rbd_mirror
         ;;
       rest_api)
-        bootstrap_rest_api
+        if [[ "$CEPH_VERSION" == "jewel" ]] || [[ "$CEPH_VERSION" == "kraken" ]] || [[ "$CEPH_VERSION" == "luminous" ]] || [[ "$CEPH_VERSION" == "mimic" ]]; then
```
Comment (ErwanAliasr1) [inline src/daemon/demo.sh]: On master, jewel/kraken are deprecated.

File: src/daemon/entrypoint.sh.in [hunk 2]
```diff
@@ -118,8 +118,13 @@ case "$CEPH_DAEMON" in
     ;;
   restapi)
     # TAG: restapi
-    source start_restapi.sh
-    start_restapi
+    if [[ "$CEPH_VERSION" == "jewel" ]] || [[ "$CEPH_VERSION" == "kraken" ]] || [[ "$CEPH_VERSION" == "luminous" ]] || [[ "$CEPH_VERSION" == "mimic" ]]; then
```
Comment (ErwanAliasr1) [inline src/daemon/entrypoint.sh.in]: Ditto

File: .travis.yml [hunk 3]
```diff
@@ -25,7 +25,7 @@ before_install:
 
 install:
   - sudo ./travis-builds/prepare_osd_fs.sh
```
Comment (ErwanAliasr1) [inline .travis.yml]: Why do you remove SREE here ?

**Comment by dsavineau** [comment]
This has been handled in https://github.com/ceph/ceph-container/pull/1364

**Pull request closed without merging**